test_suite/conftest.py

```diff
@@ -8,12 +9,21 @@
 from packaging import version
 from py.xml import html
 from pytest_html import extras
+from reportportal_client import RPLogger
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
 from lib import test_lib
 
 
+@pytest.fixture(scope="session")
+def rp_logger():
```

Comment:
I would add a if guard here, checking if CIVConfig contains `report_portal`.
If not, just return and don't continue with current logic.